pyoos/parsers/ioos/describe_sensor.py

```diff
@@ -8,7 +10,7 @@
 class IoosDescribeSensor(object):
 
     def __new__(cls, element):
-        if isinstance(element, str):
+        if isinstance(element, bytes):
```

Comment:
this should probably be `string_types`.  It's trying to detect elements from stringified elements, the `str` only bit from the py2 version should probably have been `basestring`.  I think, anyway.